collections/_article/funding-breakdown-and-hiring-process.md

```diff
@@ -0,0 +1,102 @@
+---
+title: "Godot Funding Breakdown & Hiring Process"
+excerpt: "This article aims to shed light on the current status of Godot's funding, hiring, and our plans for sustainable funding long term."
+categories: ["news"]
+author: Godot Foundation
+image: /storage/blog/covers/funding-breakdown-and-hiring-process.webp
+date: 2023-07-11 12:00:00
+---
+
+This article aims to shed light on the current status of Godot's funding, hiring, and our plans for sustainable funding long term.
+
+## Current funding and spending
+
+### State of Godot's funding
+
+Godot is sustained completely by donations. To date, those donations have taken the form of Patreon pledges, PayPal transfers, and one-time grants. By nature, only the Patreon donations are recurring (although some choose to make recurring payments through PayPal) and even then, we can't totally rely on them as the month-to-month amount fluctuates (at the time of writing it is just under $14,000 USD per month, but the amount we receive after fees is much less).
+
+To account for the temporary nature of donations, we have been cautious about expanding the team and, in the past, we waited until we had enough funds on hand to pay for a full year contract for a person before signing a contract with them.
+
+Additionally, when the project was much smaller than it is now, Godot was receiving more funds than it could use due to difficulties in finding suitable candidates for hiring and we were cautious about expanding the team. This led to Godot carrying funds over year-to-year.
```

Comment:
```suggestion
Additionally, when the project was much smaller than it is now, Godot was receiving more funds than it could use due to difficulties in finding suitable candidates for hiring. As a result, we were cautious about expanding the team. This led to Godot carrying funds over year-to-year.
```